applications/solido.md

```diff
@@ -0,0 +1,130 @@
+
+### Team Code Repos
+* https://github.com/decent-bet/solido
+* https://github.com/decent-bet/vuex-solido
+* https://github.com/decent-bet/solido-repl
+* https://github.com/decent-bet/solido-web-console
+* https://github.com/decent-bet/solido-provider-web3
+* https://github.com/decent-bet/solido-provider-thorify
+* https://github.com/decent-bet/solido-provider-connex
+* https://github.com/decent-bet/solido-cli
+* https://github.com/decent-bet/solido-block-date-converter
+* https://github.com/xdvplatform/odenplan-wallet
+* https://github.com/xdvplatform/xdvplatform-wallet
+
+### Team LinkedIn Profiles
+* https://www.linkedin.com/in/rogelio-morrell-575aa51/
+* https://www.linkedin.com/in/alfredolopez80/
+* http://linkedin.com/in/patricia-carrillo-891597172
+
+## Development Roadmap :nut_and_bolt: 
+
+PAID Network, foresees the development of Client RPC using GraphQL connectors, completely abstracts the interaction layer of the dapp with PolkaDOT/Substrate platform and the different parachains (or blockchain bridges) with the defined connector.
+
+* **Milestone 1:** Development of  Client RPC using GraphQL connectors, completely abstracts the interaction layer of the dapp with PolkaDOT/Substrate platform.
+
+### Overview
+* **Total Estimated Duration:** 3 month
+* **Full-time equivalent (FTE):**  Workload of an employed person ([see](https://en.wikipedia.org/wiki/Full-time_equivalent)) 
+* **Total Costs:** 25K
+* **BTC Address:** 3QcdnKnCAGnHRJJPQuBMmvQmJZY6GoT78C
+
+### Milestone 1 Example — Implement Plasm Modules 
+* **Estimated Duration:** 3 month
+* **FTE:**  1
+* **Costs:** 0.8 BTC
+
+| Number |   Deliverable   | Specification |
+| ------------- | ------------- | ------------- |
+| 0a. | License | Apache 2.0 / MIT / Unlicensed |
+| 0b. | Documentation | We will provide both inline documentation of the code and a basic tutorial that explains how a user can (for example) spin up one of our Substrate Runtime Module and Plasm VM's. It will be possible to send and call test transactions and meta transactions that will show how the new functionality works. |
+| 0c. | Testing Guide | The code will have proper unit-test coverage of 95% to ensure functionality and robustness. In the guide we will describe how to run these tests, And after this a Bounty policy for error detection. | 
+| 0d. | Article/Tutorial | We will write an article or tutorial that explains the work done as part of the grant. |
+| 1. | TypeScript module: | Development of templates for using graphQL as a tool to interact through these tools with each of the RPC interfaces of the different parachains that make life in polkaDOT, which in a first phase will allow the management of schema and mutations, using a single client to multiple GraphQL API's. |
```

Comment:
any rosetta openapi implementation (btc, ethereum, vechain, kadena, ...)

Reply:
In your document you should specify which **concrete** chains/parachains will be available for interaction at the time of delivery of your milestone. That forms part of the contract and will later allow us to check whether you fulfilled the necessary conditions.

Reply:
@mmagician @Noc2 let us know anything left. right now api is for dapp development, not sure if relay chains are required

Reply:
eg paid needs to be migrated to Plasm, we're going to use Chainsafe to bridge to ethereum and Plasm with ink!, without Solido Substrate, because there is no universal adapter, we'll need to migrate ethereum dapp specific code to PolkadotJS. If we had tools like the proposed dapp development will be more faster to deliver

Reply:
aditionally, I had totally forgot, there are only two known ways to create native mobile dapps: using native wallet technology like Trust Wallet Core or use GraphQL.

With Solido Substrate, which is graphql client, both querying and sending signed transactions are possible without having to build complex wallet logic, only requirement is that the wallet manages the keys in a secure way and is able to sign tx for a specific chain

Reply:
@Noc2 @enfipy @trevelyan ^^^^^